packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/schema.ts

```diff
@@ -40,14 +40,16 @@ export const QueryByParameterSchema = z.object({
         semanticsText: z.literal("LivingSubject.id"),
       })
       .optional(),
-    livingSubjectName: z.object({
-      value: schemaOrArray(nameSchema),
-      semanticsText: z.literal("LivingSubject.name"),
-    }),
+    livingSubjectName: schemaOrArray(
+      z.object({
+        value: nameSchema,
+        semanticsText: z.literal("LivingSubject.name").optional(),
+      })
+    ),
     patientAddress: z
       .object({
         value: schemaOrArrayOrEmpty(addressSchema).optional(),
-        semanticsText: z.literal("Patient.addr"),
+        semanticsText: z.literal("Patient.addr").optional(),
```

Comment:
`{
    "received": "Patient.address",
    "code": "invalid_literal",
    "expected": "Patient.addr",
    "path": [
        "Envelope",
        "Body",
        "PRPA_IN201305UV02",
        "controlActProcess",
        "queryByParameter",
        "parameterList",
        "patientAddress",
        "semanticsText"
    ],
    "message": "Invalid literal value, expected \"Patient.addr\""
}`

seems some people dont provide. no need to make this req